pages/common/netstat.md

```diff
@@ -30,4 +30,4 @@
 
 - List listening TCP and UDP ports (+ user and process if you're root):
 
-`netstat --listening --program --numeric --tcp --udp --extend`
+`netstat {{-tulpne|--listening --program --numeric --tcp --udp --extend}}`
```

Comment:
```suggestion
`netstat {{[-lpntue|--listening --program --numeric --tcp --udp --extend]}}`
```
This is the format for [option placeholders](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#option-syntax). Also keep the short option order the same as the long options.

Reply:
When was this discussed (the use of []) 
I can't see any discussion for it, and the PR is just 2 reviews without discussion.

Reply:
Sorry for my ignorance!